docker/BUILD

```diff
@@ -71,7 +82,7 @@ debug_docker_build(
 )
 
 debug_docker_build(
-    entrypoint = ["/usr/local/bin/pilot"],
+    entrypoint = ["/usr/local/bin/agent"],
```

Comment:
Bazel + go requires that cmd/agent be the binary name. cmd/istio-envoy-agent is rather long name for a binary.